test/e2e/common/tektonchainsdeployment_test.go

```diff
@@ -0,0 +1,93 @@
+// +build e2e
+
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"os"
+	"testing"
+
+	"github.com/tektoncd/operator/test/client"
+	"github.com/tektoncd/operator/test/resources"
+	"github.com/tektoncd/operator/test/utils"
+)
+
+// TestTektonChainsDeployment verifies the TektonChains creation, deployment recreation, and TektonChains deletion.
+func TestTektonChainsDeployment(t *testing.T) {
+	clients := client.Setup(t)
+
+	crNames := utils.ResourceNames{
+		TektonConfig:    "config",
+		TektonPipeline:  "pipeline",
+		TektonChains:    "chains",
+		TargetNamespace: "tekton-pipelines",
+		ChainsNamespace: "tekton-chains",
+	}
+
+	if os.Getenv("TARGET") == "openshift" {
+		crNames.TargetNamespace = "openshift-pipelines"
```

Comment:
earlier i saw `tekton-chain` ns and now in e2e its `openshift-pipelines` 
which one we need to use :thinking:

Reply:
here `targetNamespace` if for pipeline and `chainsNamespace` is for chains in e2e